src/org/usfirst/frc/team467/robot/Robot.java

```diff
@@ -46,7 +46,7 @@ public void robotInit() {
 		Logging.init();
 
 		// Initialize RobotMap
-		RobotMap.init(RobotID.Competition_2);
+		RobotMap.init(RobotID.Competition_1);
```

Comment:
Don't merge this change into master. The master branch should always be Competition_2.